app/views/category/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container">
+  <% if @categories.empty? %>
+    <p>Category lists is empty, please add a new category to begin.</p>
+  <% else %>
+    <% @categories.each do |category| %>
+      <%= link_to expense_index_path({ :category_id => category.id }), class: 'cat_id' do %>
+
+        <div class="card mb-4">
+          <div class="card-body d-flex mx-4" style="align-items:center;justify-content:space-between">
+            <img class="icon-img" src="<%= category.icon %>" alt="<%= category.name%> image">
+            <div>
+              <h5 class="card-title m-0"> <%= category.name.capitalize %> </h5>
+              <p class="card-text time"><%= category.created_at.strftime('%a %d %b, %Y')%> </p>
+            </div>
+            <h5 class="card-body m-6"> <%= number_to_currency(category.expenses.sum(:amount), unit:"৳") %> </h5>
+
+          </div>
+
+        </div>
+      <% end %>
+    <% end %>
+    </div>
+  <% end %>
+  <div class="category-div">
+    <%= link_to 'Add new Category', new_user_category_path(current_user), class: "btn new-category" %>
+  </div>
```

Comment:
Since this project is about building a mobile web application where you can manage your budget: 🤗

I noticed that your design is not mobile responsive 👀
![image](https://github.com/shihabrafio/Budget_App/assets/78251927/a15d0568-df36-4f80-bc10-c378dba99fcb)
![image](https://github.com/shihabrafio/Budget_App/assets/78251927/6af5e0df-eec8-40e6-b6e5-9b5e64e537a0)


Please follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), 😊 including:

* Colors
* Typography: font face, size, and weight.
* Layout: composition and space between elements.

Design
![image](https://github.com/shihabrafio/Budget_App/assets/78251927/795764e9-4292-4b4f-bf9e-06db7eb25b76)

Reply:
- [ ] Like the previous reviewer said, please make sure this page looks similar to the original design You should follow the layout, the colors and typography. Yours looks really different from the original design.

### Your implementation
![image](https://github.com/shihabrafio/Budget_App/assets/45460192/a1d7908d-c4b4-4280-882a-6cc0bf2e75c3)

Reply:
### [OPTIONAL]
- The same goes for your add new expense page. It does not look like that in the original design
![image](https://github.com/shihabrafio/Budget_App/assets/45460192/6a7b34d8-b198-4fe7-8cfd-481a32b41ecb)